livox_ros_driver/launch/livox_hub.launch

```diff
@@ -14,23 +14,26 @@
 	<arg name="imu_msg_frame_id" default="livox_frame"/>
 	<arg name="lidar_bag" default="true"/>
 	<arg name="imu_bag" default="true"/>
+	<arg name="angular_velocity_cov" default="[1.44e-4, 0, 0, 0, 1.44e-4, 0, 0, 0, 1.44e-4]"/>
+	<arg name="linear_acceleration_cov" default="[9.604e-3, 0, 0, 0, 9.604e-3, 0, 0, 0, 9.604e-3]"/>
 
-	<param name="xfer_format" value="$(arg xfer_format)"/>
-	<param name="multi_topic" value="$(arg multi_topic)"/>
-	<param name="data_src" value="$(arg data_src)"/>
-	<param name="publish_freq" type="double" value="$(arg publish_freq)"/>
-	<param name="output_data_type" value="$(arg output_type)"/>
-	<param name="cmdline_str" type="string" value="$(arg bd_list)"/>
-	<param name="cmdline_file_path" type="string" value="$(arg lvx_file_path)"/>
-	<param name="user_config_path" type="string" value="$(find livox_ros_driver)/config/livox_hub_config.json"/>
-	<param name="lidar_frame_id" type="string" value="$(arg lidar_msg_frame_id)"/>
-	<param name="imu_frame_id" type="string" value="$(arg imu_msg_frame_id)"/>
-	<param name="enable_lidar_bag" type="bool" value="$(arg lidar_bag)"/>
-	<param name="enable_imu_bag" type="bool" value="$(arg imu_bag)"/>
-
-	<node name="livox_lidar_publisher" pkg="livox_ros_driver"
-	      type="livox_ros_driver_node" required="true"
-	      output="screen" args="$(arg cmdline_arg)"/>
+	<node name="livox_lidar_publisher" pkg="livox_ros_driver" type="livox_ros_driver_node"
+		  required="true" output="screen" args="$(arg cmdline_arg)">
+		<param name="xfer_format" value="$(arg xfer_format)"/>
+		<param name="multi_topic" value="$(arg multi_topic)"/>
+		<param name="data_src" value="$(arg data_src)"/>
+		<param name="publish_freq" type="double" value="$(arg publish_freq)"/>
+		<param name="output_data_type" value="$(arg output_type)"/>
+		<param name="cmdline_str" type="string" value="$(arg bd_list)"/>
+		<param name="cmdline_file_path" type="string" value="$(arg lvx_file_path)"/>
+		<param name="user_config_path" type="string" value="$(find livox_ros_driver)/config/livox_hub_config.json"/>
+		<param name="lidar_frame_id" type="string" value="$(arg lidar_msg_frame_id)"/>
+		<param name="imu_frame_id" type="string" value="$(arg imu_msg_frame_id)"/>
+		<param name="enable_lidar_bag" type="bool" value="$(arg lidar_bag)"/>
+		<param name="enable_imu_bag" type="bool" value="$(arg imu_bag)"/>
+		<param name="angular_velocity_cov" type="double" value="$(arg angular_velocity_cov)"/>
```

Comment:
does this work with the array? I would have thought that the `type="double"` would expect a single value.